test/e2e/lib/components/shopping-cart-widget-component.js

```diff
@@ -43,4 +43,31 @@ export default class ShoppingCartWidgetComponent extends AsyncBaseContainer {
 			}
 		}
 	}
+
+	async removeDomainRegistraion( domain ) {
+		return this.remove( 'domain_reg', domain );
+	}
+
+	async removeDomainMapping( domain ) {
+		return this.remove( 'domain_map', domain );
+	}
+
+	async remove( type, name ) {
+		const cartBadgeLocator = by.css( '.cart__count-badge' );
+
+		const present = await driverHelper.isElementLocated( this.driver, cartBadgeLocator );
+		if ( present ) {
+			await this.open();
+			await driverHelper.clickWhenClickable(
+				this.driver,
+				by.xpath(
+					// Find an element X with class=.cart-item
```

Comment:
This is a good resource, in case you want to dive deeper into this xpath: https://devhints.io/xpath